### ISSUE TITLE
Problems with GPU partitions for VS Code 

### ISSUE THREAD

**Issue by krometis** [AdvancedResearchComputing/ARC-ondemand-apps#20]
The VS Code IDE [link](https://ood.arc.vt.edu/pun/sys/dashboard/batch_connect/sys/bc_vt_codeserver/session_contexts/new) gives users the option to choose a partition. We just had a user in office hours who requested `a100_normal_q` but the job never started because he couldn't request a GPU (`--gres=gpu:1`) and a GPU is required to be part of the request on that partition. He wanted to run VS Code there because it apparently has an integrated terminal that he figured he would use to run. I told him to do VS Code on `interactive_q` and then do his A100 job in a separate terminal, which he seemed fine with, but it seems like we should either:
- Not give users the ability to specify queue for VS Code (and perhaps other IDEs) -- i.e., automatically send requests to `interactive_q` for TC, `p100_dev_q` for Infer, etc, OR
- Give them the ability to supply additional flags to the request so the jobs can start

Also, the VS Code app mentions V100s:
> To request a GPU enabled queue, preface it with v100_. Example: v100_normal_q, t4_normal_q

which is of course not applicable to newer clusters.

**Comment by rsettlage** [comment]
Ok, so I can add the `gpu` box, but if the interactive queue thing works, that seems like a better way to go.  

As to the `v100` etc, this can run on Infer, so those partitions are valid, but perhaps not optimal.  ??  

Ultimately, I guess the question is if there is value on running this on the node type you are dev'ing on?

**Comment by rsettlage** [comment]
After speaking with a couple others on how they use VS Code, I changed the partition note to:

> Note: do not select a GPU queue, if you are developing interactively, start a second job containing the desired resources t
o actually run your code.

I was going to provide a queue list to choose from, but decided to hold off until we discussed how we envision this on Infer.  Ideally, we would simplify this form to just use interactive queues, but we don't currently have one on Infer.
Closing.
